index.html

```diff
@@ -94,11 +94,151 @@ <h2>Software Developer</h2>
           <li></li>
         </ul>
 
-        <div class="work">
+        <div class="work-direction">
           <img src="/images/Arrow.png" alt="down-arrow" />
           <span>Work</span>
         </div>
       </div>
+
+      <!-- Work -->
+
+      <section class="work">
+        <div id="work-title">
+          <h2>MY RECENT <span id="break">WORK</span></h2>
+          <img src="/images/vector-line.png" alt="line" />
+        </div>
+
+        <ul class="all_projects">
+          <li class="project">
```

Comment:
- [ ] You did a great job displaying the different works section using `display:grid`, however, as per requirement, each single work card should have its content displayed using Flexbox.
Hint: use `display:flex` with a flex-flow of column.